### DISCUSSION TITLE
Help me, I install tailwind v3 and upgrade last version typogrphy and form, but receive error 700 of undefine,

### DISCUSSION THREAD

**Discussion by dbstrieu** [tailwindlabs/tailwindcss#11409] [Help]
Help me, I install tailwind v3 and upgrade last version typogrphy and form, but receive error 700 of undefined.
I try fix in 3 days but not done. 
In file package lock.json:
"@tailwindcss/forms": "^0.5.3",
 "@tailwindcss/typography": "^0.5.9",
 "tailwindcss": "^3.3.2",
 "tailwindcss-animate": "^1.0.5",

**Comment by wongjn** [comment]
Could you give more details that reproduce the problem you are experiencing please? Perhaps a remotely-hosted git repository along with steps to reproduce the error.

**Reply by dbstrieu** [reply]
hi @wongjn 
Cannot read properties of undefined (reading '700') at \node_modules\@tailwindcss\typography\src\styles.js 
when there are variables declaring square brackets of type.gray[700] in \node_modules\@tailwindcss\typography\src\styles.js
This when I installed it was the default of tailwind v3.
If I delete the reuquire typography in the plugin it will get rid of the error but I want to keep the typogrphy for use.

**Reply by wongjn** [reply]
Thank you! Please consider providing a remotely-hosted git repository along with steps to reproduce the error.

**Comment by mybearworld** [comment]
Try running `npm update`

**Reply by chandragaajula** [reply]
This doesn't help. Please recommend other steps.

**Reply by wongjn** [reply]
Consider providing a project that reproduces the error.

**Reply by chandragaajula** [reply]
Thanks for the quick response @wongjn 

I have created a new discussion thread with all the details: https://github.com/tailwindlabs/tailwindcss/discussions/12973
